src/util/change-types.ts

```diff
@@ -124,7 +124,6 @@ export const changeLineStyleType = (
     const newAttrs = structuredClone(lineStyles[newLineStyleType].defaultAttrs);
     if (LineStylesWithColor.includes(currentLineStyleType) && LineStylesWithColor.includes(newLineStyleType))
         (newAttrs as AttributesWithColor).color = (oldAttrs as AttributesWithColor).color;
-    else if (newLineStyleType === LineStyleType.SingleColor && theme) (newAttrs as SingleColorAttributes).color = theme;
```

Comment:
Hi, I believe this is used when the user is changing the style to the single color style and the current theme will be applied automatically.

Reply:
L126 has done it 👀

Reply:
Oh, It's my fault. I'll put it back.